build/bundle.js

```diff
@@ -21098,6 +21086,349 @@ var FCC_Global =
+
+	/**
+	 * Compare feature coordinate to tick[i] and tick[i+1] and return true if
+	 * feature coordinate is greater than or equal to the coordinate for tick[i]
+	 * and less than or equal to the coordinate for tick[i+1]
+	 * @function
+	 * @param {Number} featureCoord - x or y coordinate per feature per axis
+	 * @param {Object} axis - Object constructed from getAxisInfo
+	 * @param {Number} tickPxCur - Coordinate (x or y) for tick[i]
+	 * @param {Number} tickPxNext - Coordinate (x or y) for tick[i+1]
+	 * @returns {Boolean} true if feature position is between (inclusive) tick[i]
+	 * and tick[i+1]
+	 */
+	var isFeatureAligned = function isFeatureAligned(featureCoord, axis, tickPxCur, tickPxNext) {
+	  // Determine lower / higher coordinate size between two ticks
+	  // This is to account for reverse tick order (see issue #70)
+	  var minTickCoord = Math.min(tickPxCur, tickPxNext);
+	  var maxTickCoord = Math.max(tickPxCur, tickPxNext);
+	  // getMisalignmentCount checked whether the feature's attribute data-xvalue or
+	  // data-yvalue is between (inclusive) two ticks' labeled values.
+	  // If that is true, the feature's pixel coordinates should also be between
+	  // (inclusive) the two ticks' axis coordinates.
+	  return featureCoord >= minTickCoord && featureCoord <= maxTickCoord;
+	};
+
+	/**
+	 * For given tick i and i+1, get features whose associated values are
+	 * between (inclusive) tick i and i+1 values and runs isFeatureAligned on each.
+	 * @param {Number} tickPxCur - Coordinate (x or y) for tick[i]
+	 * @param {Number} tickPxNext - Coordinate (x or y) for tick[i+1]
+	 * @param {Number} tickValCur - parsed from getTickValue(getTickText())
+	 * @param {Number} tickValNext - parsed from getTickValue(getTickText())
+	 * @param {Object} axis - object created from getAxisInfo
+	 * @param {Array} collection - HTMLCollection of features
+	 * @param {Function} getFeatureValue - (x or y) via getMisalignmentCountCaller
+	 * @param {Function} getFeatureCoord - (x or y) via getMisalignmentCountCaller
+	 * @return {Number} Count of misaligned features via isFeatureAligned
+	*/
+	var getMisalignmentCount = function getMisalignmentCount(tickPxCur, tickPxNext, tickValCur, tickValNext, axis, collection, getFeatureValue, getFeatureCoord) {
+	  var count = 0;
+
+	  for (var j = 0; j < collection.length - 1; j++) {
+	    // get values for given feature (j)
+	    var featureVal = getFeatureValue(collection.item(j)),
+	        featureCoord = getFeatureCoord(collection.item(j));
+	    if (featureVal >= tickValCur && featureVal <= tickValNext) {
+	      if (!isFeatureAligned(featureCoord, axis, tickPxCur, tickPxNext)) {
+	        count++;
+	      }
+	    }
+	  }
+	  return count;
+	};
+
+	/**
+	 * Get x coordinate of given feature
+	 * @function
+	 * @param {String} feature - HTMLElement circle (.dot)
+	 * @returns {Number} float
+	 */
+	var getXFeatureCoord = function getXFeatureCoord(feature) {
+	  return parseFloat(feature.getAttribute('cx') - 0.5);
```

Comment:
Have you tested this locally too? If it passes locally and with CodePen then this is ready for merge. Let me know when you are around because I don't want to merge it until we are both available to test it.